.github/workflows/main.yml

```diff
@@ -7,9 +7,9 @@ on:
     tags:
     - v*
   pull_request:
-    branches:
-    - dev
-    - master
+    types:
+    - synchronize
+    - ready_for_review
```

Comment:
This is interesting. What do these mean? I still think it might make sense to run the CI on PRs that aren't ready for review, for example.

Reply:
I changed this because I noticed the workflow was not running for PRs that I had opened with a base that was not `dev` or `master`.

I got a explanation from their docs: 

`synchronize`: Triggered when a pull request's head branch is updated. For example, when the head branch is updated from the base branch, when new commits are pushed to the head branch, or when the base branch is changed. (https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request)

`ready_for_review`: A draft pull request was marked as ready for review. (https://docs.github.com/en/developers/webhooks-and-events/events/issue-event-types#ready_for_review)

But I'm not sure if `synchronize`  works for draft PRs.

Reply:
There are some people trying workarounds to avoid running jobs in Draft PRs, so I guess they actually run in Draft PRs as well with this config.

Example of discussion on workarounds: https://github.community/t/dont-run-actions-on-draft-pull-requests/16817/18

Reply:
So it looks like the workflow of PRs in draft might run because of `synchronize` then. This is what I prefer actually: to not prevent workflows from running just because they are drafts. But now I'm not sure what `ready_for_review` adds on top of `synchronize`, wouldn't `synchronize` be enough?

Reply:
Done

Reply:
Great!